src/maps/mod.rs

```diff
@@ -335,10 +335,7 @@ impl<'a> PerfEventIterator<'a> {
 
         #[cfg(feature = "metrics")]
         {
-            let labels = [
-                ("map_name", map.name.clone()),
-                ("cpu", map.cpuid.to_string()),
-            ];
+            let labels = [("map_name", map.name.clone())];
```

Comment:
I can't imagine this broke doctests, so I'm good to blame CircleCI for this one.